README.md

```diff
@@ -139,75 +139,137 @@ tool for the simulation and drive the vehicle towards a wall.
 
 Note the following topic names for your publishers and subscribers:
 
-*xxx-ct Breanna, the Odometry entry in the list below seems*
-*suspicious. Should that be "`Odometry`: /ego_racecar, /odom"?*
-*please correct it.*
-
 - `LaserScan`: /scan
 - `Odometry`: /ego_racecar/odom, specifically, the longitudinal velocity of the vehicle can be found in `twist.twist.linear.x`
 - `AckermannDriveStamped`: /drive
 
-## 5. Deliverables and Submission
-
-You can implement this node in either C++ or Python. A skeleton
-package is already provided in the repo that you can use.
-
-*xxx-ct Breanna, please provide a description of how the student would
-copy (or place) the skeleton in the correct location. The exact
-commands would be most helpful:*
+## 5. Package
+
+You can implement this node in either C++ or Python. Clone the 
+f1tenth open repository outside of the f1tenth
+Gym container directory created in the previous lab.
+You will need to reference this section for use in future labs to 
+add them into your directory.
+
+```bash
+# On the localhost
+gordon@f1sim:~$ mkdir git
+gordon@f1sim:~$ cd git/
+gordon@f1sim:~/git$ git clone https://github.com/f1tenth/f1tenth_labs_openrepo.git
+gordon@f1sim:~/git$ cd f1tenth_labs_openrepo/
+gordon@f1sim:~/git/f1tenth_labs_openrepo$ git checkout main 
+```
 
-   PLACEHOLDER
-   FOR
-   EXACT
-   COMMANDS
+Copy the skeleton into the same directory containing the gym workspace.
+```bash
+# On the localhost
+gordon@f1sim:~$ cd sim_ws/src/
+gordon@f1sim:~sim_ws/src/$ cp -r ~/git/f1tenth_labs_openrepo/f1tenth_lab2/safety_node .
```

Comment:
They should clone just the f1tenth_lab2_template from my account, which your changes will be pushed to.